cdap-security/src/main/java/co/cask/cdap/security/authorization/DefaultAuthorizationEnforcer.java

```diff
@@ -60,6 +63,12 @@ public void enforce(EntityId entity, Principal principal, Action action) throws
   }
 
   private void doEnforce(EntityId entity, Principal principal, Set<Action> actions) throws Exception {
+    if (entity instanceof NamespacedEntityId &&
```

Comment:
Oh, my bad, it should only bypass if the principal is the master user, will make the change